src/Utils/messages.ts

```diff
@@ -322,7 +322,8 @@ export const generateWAMessageContent = async(
 			buttonText: message.buttonText,
 			title: message.title,
 			footerText: message.footer,
-			description: message.text
+			description: message.text,
+			listType: message.listType
```

Comment:
```suggestion
			listType: 1
```

to keep everything simple, just make it default to 1. Now there is no need to modify existing code